### PR TITLE
Edit build

### LINKED ISSUE #3: Make Building Work
Fix build

### PR DESCRIPTION
**Pull request by Sphyralg**
revert a commit that prevents the build from working

fixes #3

### CODE REVIEW

**Comment by zx96** [comment]
It looks like `scripts.inc` gets built from `scripts.pory` for the maps - do we just need to remove and add `scripts.inc` to a `.gitignore` file to fix this?

**Comment by Sphyralg** [comment]
I will investigate and see if the .inc files can be removed from the git tracking to generate a valid build

File: Makefile [hunk 1]
```diff
@@ -165,7 +165,7 @@ MID_OBJS := $(patsubst $(MID_SUBDIR)/%.mid,$(MID_BUILDDIR)/%.o,$(MID_SRCS))
 OBJS := $(C_OBJS) $(C_ASM_OBJS) $(ASM_OBJS) $(DATA_ASM_OBJS) $(SONG_OBJS) $(MID_OBJS)
 OBJS_REL := $(patsubst $(OBJ_DIR)/%,%,$(OBJS))
 
-TOOLDIRS := $(filter-out tools/agbcc tools/binutils tools/analyze_source,$(wildcard tools/*))
+TOOLDIRS := $(filter-out  tools/poryscript tools/agbcc tools/binutils tools/analyze_source,$(wildcard tools/*))
```
Comment (zx96) [inline Makefile:168]: Where does Poryscript come from? Do we need to add it ourselves to build?

Comment (zx96) [reply Makefile:168]: Sounds like yes - we should probably update the README to mention this.

**Pull request merged**